.github/workflows/check.yml

```diff
@@ -9,23 +9,23 @@ on:
 jobs:
   build:
     name: "Checking the package: testing and linting"
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     strategy:
       matrix:
         php: ["8.2", "8.3"]
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4.2.2
```

Comment:
Why not to add Renovate? We can limit managers to github actionsonly.

Reply:
I can add config. I just didn't ask @krzysztofrewak about adding Renovate to this repository.